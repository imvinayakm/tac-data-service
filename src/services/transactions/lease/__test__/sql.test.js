const sql = require('../sql');
const filterValues = {
  recipient: 'recipient',
};

describe('Sql search by type-specific filters', () => {
  it('supports recipient filter', () => {
    expect(sql.search(filterValues)).toMatchSnapshot();
  });
});
