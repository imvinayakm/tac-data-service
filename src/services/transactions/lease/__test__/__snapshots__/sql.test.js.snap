// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sql search by type-specific filters supports recipient filter 1`] = `"select \\"t\\".\\"height\\" as \\"height\\", \\"t\\".\\"tx_type\\" as \\"tx_type\\", \\"t\\".\\"id\\" as \\"id\\", \\"t\\".\\"time_stamp\\" as \\"time_stamp\\", \\"t\\".\\"signature\\" as \\"signature\\", \\"t\\".\\"proofs\\" as \\"proofs\\", \\"t\\".\\"tx_version\\" as \\"tx_version\\", t.fee * 10^(-8) as \\"fee\\", \\"t\\".\\"sender\\" as \\"sender\\", \\"t\\".\\"sender_public_key\\" as \\"sender_public_key\\", t.amount * 10^(-8) as \\"amount\\", \\"t\\".\\"recipient\\" as \\"recipient\\" from \\"txs_8\\" as \\"t\\" where \\"recipient\\" = 'recipient'"`;
